<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordSummaryDialog</class>
 <widget class="QDialog" name="RecordSummaryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QWidget" name="dialogContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>776</width>
       <height>348</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>776</width>
       <height>625</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#dialogContainer {
	border-radius: 12px;
	background-color: white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="headerRow" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="styleSheet">
            <string notr="true">color: black;
font-family: Open Sans;
font-weight: 500;
font-size: 22px;
line-height: 30px;
letter-spacing: 0%;
</string>
           </property>
           <property name="text">
            <string>Software Testing Mechanism</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="RecIDPart" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="IdLabel">
              <property name="styleSheet">
               <string notr="true">QLabel {
font-family: Open Sans;
font-weight: 500;
font-size: 18px;
line-height: 26px;
letter-spacing: 0%;
color: black;
}</string>
              </property>
              <property name="text">
               <string>Id:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="recordIDLabel">
              <property name="styleSheet">
               <string notr="true">QLabel {
font-family: Open Sans;
font-weight: 500;
font-size: 16px;
line-height: 24px;
letter-spacing: 0%;
color: #666666
}</string>
              </property>
              <property name="text">
               <string>#WTGH234</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="expendWindowButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../../Resources/Resources.qrc">
                <normaloff>:/images/Project/expend-window.svg</normaloff>:/images/Project/expend-window.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../Resources/Resources.qrc">
             <normaloff>:/images/Project/cross-icon.svg</normaloff>:/images/Project/cross-icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="summaryGroup" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#summaryGroup {
	background-color:transparent;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>34</number>
         </property>
         <item>
          <widget class="QWidget" name="fieldRow1" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>68</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="typeField" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="typeHeader" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 20px;
line-height: 28px;
letter-spacing: 0%;
}</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="maximumSize">
                     <size>
                      <width>32</width>
                      <height>32</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="pixmap">
                     <pixmap resource="../../../../Resources/Resources.qrc">:/images/Record/type-icon.svg</pixmap>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="typeLabel">
                    <property name="text">
                     <string>Type</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="typeFieldLabel">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
color: #4D4D4D;
font-family: Open Sans;
font-weight: 500;
font-size: 18px;
line-height: 26px;
letter-spacing: 0%;
}</string>
                 </property>
                 <property name="text">
                  <string>System Requirement</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="versionField" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="versionHeader" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 20px;
line-height: 28px;
letter-spacing: 0%;
}</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="versionIcon">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="pixmap">
                     <pixmap resource="../../../../Resources/Resources.qrc">:/images/Record/version-icon.svg</pixmap>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="versionLabel">
                    <property name="text">
                     <string>Version</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="versionFieldLabel">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
color: #4D4D4D;
font-family: Open Sans;
font-weight: 500;
font-size: 18px;
line-height: 26px;
letter-spacing: 0%;
}</string>
                 </property>
                 <property name="text">
                  <string>2.0.8.9.1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="statusField" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="statusHeader" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 20px;
line-height: 28px;
letter-spacing: 0%;
}</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="statusIcon">
                    <property name="maximumSize">
                     <size>
                      <width>32</width>
                      <height>32</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="pixmap">
                     <pixmap resource="../../../../Resources/Resources.qrc">:/images/Record/status-icon.svg</pixmap>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="statusHeaderLabel">
                    <property name="text">
                     <string>Status</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="statusValueLabel">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
color: #4D4D4D;
font-family: Open Sans;
font-weight: 500;
font-size: 18px;
line-height: 26px;
letter-spacing: 0%;
}</string>
                 </property>
                 <property name="text">
                  <string>Draft</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="descField" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="descHeader" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <property name="spacing">
                <number>8</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="descIcon" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/images/Record/desc-icon.svg);</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="descLabel">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 20px;
line-height: 28px;
letter-spacing: 0%;
}</string>
                 </property>
                 <property name="text">
                  <string>Description</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="descTextView">
              <property name="styleSheet">
               <string notr="true">QTextBrowser {
font-family: Open Sans;
font-weight: 500;
font-size: 18px;
line-height: 26px;
letter-spacing: 0%;
color: #4D4D4D;
background-color: rgb(255, 255, 255);
}

QScrollBar {
    background: transparent;
}

QScrollBar:vertical {
    width: 6px;
    background: transparent;
}
QScrollBar::handle:vertical {
    background: #4B286D;
    border-radius: 3px;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0;
}</string>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Open Sans'; font-size:18px; font-weight:500; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:18pt; font-weight:400;&quot;&gt;Lorem ipsum dolor sit amet consectetur. Sapien fermentum viverra ut nulla. Id condimentum enim at ullamcorper dolor purus morbi. Senectus sapien tempor sem viverra. Dictum porttitor enim viverra sed accumsan. Arcu neque est sed quisque feugiat nisl ac a. Ac dolor viverra massa luctus diam. Ac amet malesuada sodales dignissim turpis non in nulla at. Consectetur eget gravida magnis porta nullam in malesuada facilisis. Vestibulum commodo dui posuere nascetur cras neque sed. Vestibulum pellentesque est nulla pretium in tempus sed platea. Nam imperdiet vitae turpis in cras. Pretium integer feugiat a at risus. Nec sollicitudin suspendisse tellus See more&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordSummaryDialog</receiver>
   <slot>onCloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCloseButtonClicked()</slot>
 </slots>
</ui>
